cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -603,6 +568,64 @@ public void close() throws IOException {
     }
   }
 
+  /**
+   * Generates a SELECT query for scanning over all the provided ranges. For each of the range, it
+   * generates a where clause using the {@link #appendRange(StringBuilder, Range)} method. The where
+   * clause of each range are OR together. E.g.
+   *
+   * <p>SELECT * FROM table WHERE (key1 = ? AND key2 = ?) OR (key1 = ? AND key2 = ?) OR ((key3 >= ?)
+   * AND (key3 <= ?)) OR ((key4 >= ?) AND (key4 <= ?)) LIMIT limit
+   *
+   * @param singletonRanges the list of singleton ranges to scan
+   * @param ranges the list of ranges to scan
+   * @param limit number of result
+   * @return a select query
+   */
+  private PreparedStatement prepareMultiScanQuery(Collection<Range> singletonRanges,
```

Comment:
are there any logical changes being made here or is it just getting moved? (In the future, keep methods in the same location so it is easier to review the diff)

Reply:
This was done to fix a checkstyle warning about keeping all the `override` methods together. There is no logical change. I will revert it back to reduce the diff

Reply:
I see, if it is fixing checkstyle it's ok to move it.